src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -164,11 +171,71 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
   }
 
   if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    const { data: reservePositions } = await axios.get<ThorchainRunepoolReservePositionsResponse>(
+      `${getConfig().REACT_APP_MIDGARD_URL}/member/thor1dheycdevq39qlkxs2a6wuuzyn4aqxhve4qxtxt`,
+    )
     const { data: runepoolInformation } =
       await axios.get<ThorchainRunepoolInformationResponseSuccess>(
         `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/runepool`,
       )
 
+    const { data: poolsResponse } = await axios.get<ThornodePoolResponse[]>(
+      `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/pools`,
+    )
+
+    const poolsByAsset = poolsResponse.reduce<Record<string, ThornodePoolResponse>>(
```

Comment:
suggestion: since we're already iterating over `underlyingAssetIds` above and doing `poolAssetIdToAssetId`, could make this `poolsByAssetId` and leverage AssetId-first terminology instead, but absolutely fine as-is

Reply:
Done in [1274545](https://github.com/shapeshift/web/pull/7459/commits/1274545945102b423436a25ccdf5fcfc878b6644)